packages/kit/test/apps/basics/src/routes/+layout.server.js

```diff
@@ -1,24 +1,15 @@
 import { error, redirect } from '@sveltejs/kit';
 
-let should_fail = '';
-/**
- * @param {string} value
- */
-export function set_should_fail(value) {
-	should_fail = value;
-}
-
 /** @type {import('./$types').LayoutServerLoad} */
-export async function load() {
+export async function load({ cookies }) {
+	const should_fail = cookies.get('fail-type');
 	if (should_fail) {
+		cookies.delete('fail-type');
```

Comment:
Should we have a `path: '/'` here as well if we're creating the cookie with one?

Reply:
yes

```suggestion
		cookies.delete('fail-type', { path: '/' });
```